sqlmesh/core/snapshot/evaluator.py

```diff
@@ -136,9 +136,6 @@ def apply(query_or_df: QueryOrDF, index: int = 0) -> None:
                 else:
                     raise SQLMeshError(f"Unexpected SnapshotKind: {snapshot.model.kind}")
 
-        for sql_statement in model.sql_statements:
```

Comment:
let's delete sql_statements in model definition if it is not used

Reply:
Ok I'll take a look, will do 👍